lib/api/controllers/bulkController.js

```diff
@@ -22,18 +22,32 @@
 'use strict';
 
 const
-  PartialError = require('kuzzle-common-objects').errors.PartialError,
+  BaseController = require('./controller'),
+  {errors: {PartialError}} = require('kuzzle-common-objects'),
   {assertHasBody, assertBodyHasAttribute} = require('../../util/requestAssertions');
 
 /**
  * @class BulkController
  * @param {Kuzzle} kuzzle
  */
-class BulkController {
+class BulkController extends BaseController {
   constructor (kuzzle) {
+    super(kuzzle);
```

Comment:
Why aren't you declaring the action in the `actions` property for this one?

Reply:
Because I prefer my code duplicated. DRY is for the weak.

![2wf3wi](https://user-images.githubusercontent.com/12997967/54623123-399c2000-4a6b-11e9-8b54-0afa51c2442f.jpg)


(fix incoming :expressionless: )

Reply:
Fixed.